modules/local/validation/metadata/validate_metadata.py

```diff
@@ -0,0 +1,268 @@
+            insert_size = rg.get('insert_size', '').strip()
+            if insert_size:
+                size = int(insert_size)
+                if size <= 0:
+                    validation_errors.append(f'Read group {rg_id}: insert_size must be positive integer')
+        except ValueError:
+            validation_errors.append(f'Read group {rg_id}: insert_size must be a valid integer')
+    
+    return validation_errors
+
+
+def validate_payload_analysis_type(payload):
+    """Validate analysis type structure in payload."""
+    validation_errors = []
+    
+    analysis_type = payload.get('analysisType', {})
+    if not isinstance(analysis_type, dict):
+        validation_errors.append('analysisType must be an object')
+        return validation_errors
+    
+    # Check for required analysis type fields
+    required_analysis_fields = ['name', 'version']
+    for field in required_analysis_fields:
+        if field not in analysis_type:
+            validation_errors.append(f'Missing required analysis type field: {field}')
+    
+    return validation_errors
+
+
+def basic_payload_validation(payload_file):
+    """Perform basic payload validation without read group data."""
+    try:
+        payload = load_analysis_payload(payload_file)
+        validation_errors = validate_payload_analysis_type(payload)
+        
+        if validation_errors:
+            return False, validation_errors
+        
+        print('Basic metadata validation completed successfully')
+        return True, []
+    
+    except Exception as e:
+        return False, [str(e)]
+
+
+def comprehensive_validation(payload_file, read_group_file):
+    """Perform comprehensive validation with both payload and read group data."""
+    try:
+        # Load data
+        payload = load_analysis_payload(payload_file)
+        read_groups = load_read_group_data(read_group_file)
+        
+        print(f'Loaded {len(read_groups)} read groups from biospecimen data')
+        
+        # Perform all validations
+        validation_errors = []
+        validation_errors.extend(validate_payload_analysis_type(payload))
+        validation_errors.extend(validate_read_group_fields(read_groups))
+        validation_errors.extend(validate_read_group_files(payload, read_groups))
+        
+        if validation_errors:
+            return False, validation_errors
+        
+        print('Read group file validation completed successfully')
+        print('All file references are consistent between read groups and payload')
+        return True, []
+    
+    except Exception as e:
+        return False, [str(e)]
+
+
+def main():
+    """Main function for command line execution."""
+    parser = argparse.ArgumentParser(description='Validate metadata consistency between payload and read group data')
+    parser.add_argument('payload_file', help='Path to analysis payload JSON file')
+    parser.add_argument('--read-group-file', help='Path to read group TSV file (optional)')
```

Comment:
set default to false so that in the try it'll be evaluated a boolean when deciding comprehensive vs basic validation. Otherwise i'm confused when `args.read_group_file` and `os.path.exists(args.read_group_file)` would not co-exist?